tests/data_files/Makefile

```diff
@@ -49,18 +49,32 @@ all_intermediate += test-ca.req.sha256
 
 test-ca.crt: $(test_ca_key_file_rsa) test-ca.req.sha256
 	$(MBEDTLS_CERT_WRITE) is_ca=1 serial=3 request_file=test-ca.req.sha256 selfsign=1 issuer_name="C=NL,O=PolarSSL,CN=PolarSSL Test CA" issuer_key=$(test_ca_key_file_rsa) issuer_pwd=$(test_ca_pwd_rsa) not_before=20110212144400 not_after=20210212144400 md=SHA1 version=3 output_file=$@
-test-ca.der: test-ca.crt
+all_final += test-ca.crt
```

Comment:
None of the changes in the directory `tests/data_files` are relevant. They're all changes to certificates or to keys that are only used to run tests with certificates.

Reply:
This was a particularly hairy bit of the merge. For the sake of future merges, it's good to have this match more closely with what's in Mbed TLS.

The changes came along with upstream changes that allow testing even with PEM is disabled, which is still relevant to Mbed Crypto. See upstream PR https://github.com/ARMmbed/mbedtls/pull/2260

Note that `library/certs.c` is part of libmbedcrypto.[so|a] until we do an Mbed TLS 3.0.

Reply:
Ugh, right. Even before Mbed TLS 3.0, we could move `certs.c` to mbedtls, and link it into `libmbedcrypto` when building that in mbedtls, but I guess we won't get around to doing it that way. So ok, let's keep these changes.